lib/govuk_index/publishing_event_worker.rb

```diff
@@ -36,6 +36,12 @@ def perform(messages)
       processor = Index::ElasticsearchProcessor.govuk
 
       messages.each do |routing_key, payload|
+        if payload.is_a?(Hash) && payload.has_key?("base_path")
+          base_path = payload["base_path"]
+          extra_metadata = Indexer::MetadataTagger.metadata_for_base_path(base_path)
+          payload = payload.merge(extra_metadata)
```

Comment:
In `email-alert-api` we used slightly more complex logic to try and merge arrays nicely - https://github.com/alphagov/email-alert-api/pull/731/files#diff-9959dd9022669e7af3eea3a452bb2643R13. I'm not sure if you want to add this to Rummager.

I guess this is why we might want a Gem...

Reply:
I think this is fine, because it'll only cause a problem if the content is both tagged by the publishing app and in the csv (and with different metadata in each).